wiki/models/urlpath.py

```diff
@@ -110,7 +112,7 @@ def delete_subtree(self):
     
     @classmethod
     def root(cls):
-        site = Site.objects.get_current()
+        site = get_current_site(get_current_request())
```

Comment:
Have you tested this in edx-platform? If I look at the implementation of `get_current()` which is what `get_current_site()` calls, if the `SITED_ID` is set in settings, the `request` parameter is ignored and `Site` for the `SITE_ID` is returned.

https://github.com/django/django/blob/1.8.9/django/contrib/sites/models.py#L50-L69